File: app/src/main/java/com/example/communityeventmanagementsystem/domain/model/ForumMessage.kt
package com.example.communityeventmanagementsystem.domain.model

data class ForumMessage(
    val id: Long,
    val communityId: Long,
    val senderId: Long,
    val message: String,
    val senderName: String,
    val senderAvatarUrl: String?,
    val createdAt: String
)

File: communityApp-Backend/app/Models/ForumMessage.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\SoftDeletes;

class ForumMessage extends Model
{
    use SoftDeletes;

    protected $fillable = [
        'community_id',
        'sender_id',
        'message'
    ];

    public function community()
    {
        return $this->belongsTo(
            Community::class
        );
    }

    public function sender()
    {
        return $this->belongsTo(
            User::class,
            'sender_id'
        );
    }
}